### PR TITLE
[WIPTEST]Stack namvamzing impact

### PR DESCRIPTION
**Pull request by sshveta**
Navmazing conversion impact on stack provisioning

### CODE REVIEW

File: cfme/tests/services/test_provision_stack.py [hunk 1]
```diff
@@ -341,15 +329,15 @@ def test_retire_stack(provider, provisioning, catalog, catalog_item, request):
...
+    service_catalogs = ServiceCatalogs(item_name, stack_data)
+    service_catalogs.order()
     logger.info('Waiting for cfme provision request for service %s', item_name)
     row_description = item_name
     cells = {'Description': row_description}
     row, __ = wait_for(requests.wait_for_request, [cells, True],
                        fail_func=requests.reload, num_sec=2500, delay=20)
 
-    assert provision_success_message(catalog_item.name) in row.last_message.text
+    assert 'Provisioned Successfully' in row.last_message.text
```
Comment (sshveta) [inline cfme/tests/services/test_provision_stack.py]: screenshot from 2017-01-09 18 19 54


Last message is not same for all requests.hence checking 
"Provisioned successfully " only

File: cfme/tests/services/test_provision_stack.py [hunk 2]
```diff
@@ -258,20 +244,21 @@ def _cleanup_vms():
                 wait_for(lambda: provider.mgmt.delete_stack(stack_data['stack_name']),
                          delay=10, num_sec=800, message="wait for stack delete")
             stack_data['vm_name'].delete_from_provider()
+            catalog_item.orch_template.delete()
```
Comment (sshveta) [inline cfme/tests/services/test_provision_stack.py]: Template can be deleted only after deleting stack

**Comment by sshveta** [comment]
Service name is now sometimes appended with timestamp , which is creating an issue in selecting the node in tree .Working on it now.

**Comment by dajoRH** [comment]
Lint report for commit 1209f1d0fb919f80ac5681f89aba7ecce70d1235:

`cfme/tests/services/test_provision_stack.py`:
- :red_circle: Line 12:1: **F401** *'CandidateNotFound' imported but unused*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by psav** [comment]
I'm OK to do that in another PR, I'm more keen to get navmazing stuff merged right now

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Pull request closed without merging**